app/code/Magento/InventoryAdminUi/Test/Mftf/Test/VerifyThatMessageOnly0LeftNotAppearedOnSimpleProductPageOnFrontendTest.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="VerifyThatMessageOnly0LeftNotAppearedOnSimpleProductPageOnFrontendTest">
```

Comment:
Let's avoid numerics in test names. I'd stick with a simple descriptive name. "zero" may be used instead of "0" if its needed

Reply:
ok